### DISCUSSION TITLE
Filezilla Client

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#85] [General]
# Filezilla Client

Check if Filezilla Client is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/21790c50-decf-4387-8b66-be3efebcb369

**Comment by MikeFarrington** [comment]
A distinction should be made between official native releases, and unofficial support added by third-parties such as this Filezilla client.  While the work of the porter is greatly appreciated, I think it's important to highlight that this is not officially supported and the person(s) who re-compiled and re-packaged a well-known and trusted product do not deserve the same level of trust as the official group.  

I don't think this should be pulled.  But I think the unofficial nature of the linked application (and inherent dangers) should be made clear.
